tests/core/recipes/espresso_recipes/test_phonons.py

```diff
@@ -188,3 +193,27 @@ def test_matdyn_job(tmp_path, monkeypatch):
     assert Path(recycled_input[0]["input"].pop("flfrc")).is_absolute()
 
     assert recycled_input[0]["input"] == input_data["input"]
+
+
+def test_phonon_dos_flow(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+
+    atoms = bulk("Li")
+
+    copy_decompress_files([DATA_DIR / "Li.upf.gz"], tmp_path)
+
+    SETTINGS.ESPRESSO_PSEUDO = tmp_path
```

Comment:
We want to make sure this is returned to its default value after the test is done.

So, do `DEFAULT_SETTINGS = SETTINGS.model_copy()` at the top of the module. And then at the bottom of this function do `SETTINGS.ESPRESSO_PSEUDO = DEFAULT_SETTINGS.ESPRESSO_PSEUDO`. 

[It'd be nice](https://github.com/Quantum-Accelerators/quacc/issues/1791) of there were a way to cleanly do this with `monkeypatch.setattr`, but I've struggled to do so.